spark-derived-feature-view/README.md

```diff
@@ -0,0 +1,98 @@
+# Overview
+
+This example shows how to use `DerivedFeatureView` to backfill the input dataset
+with extra features for offline training. It involves the following steps:
+
+1. Read a batch of historical purchase events from a file.
+
+   Each purchase event has the following fields:
+   - user_id, unique identifier of the user that made the purchase.
+   - item_id, unique identifier of the item that is purchased.
+   - item_count, number of items purcahsed.
+   - timestamp, time when this purchase is made.
+
+2. Read a batch of historical item price events from a file.
+
+   Each item price event has the following fields:
+   - item_id, unique identifier of the item.
+   - price, the new price of this item.
+   - timestamp, time when the new price is used for this item.
+
+3. For each purchase event, append the following two fields by joining with item
+   price events and performing over-window aggregation, with point-in-time
+   correctness in both operations.
+
+   - price, price of the item at the time this purchase is made.
+   - total_payment_last_two_minutes, total cost of purchases made by this
+     user in a 2-minute window that ends at the time this purchase is made.
+
+4. Output the batch of purchase events backfilled with the extra features to a
+   file.
+
+
+# Prerequisites
+
+Prerequisites for running this example:
+- Unix-like operating system (e.g. Linux, Mac OS X)
+- Python 3.7
+- Docker Compose >= 20.10
+
+# Step-By-Step Instructions
+
+Please execute the following commands under the `spark-derived-feature-view`
+folder to run this example.
+
+1. Build the Spark image with Feathub installed.
+
+   ```bash
+   $ docker build --rm -t feathub-spark:3.3.1 -f ../docker/Dockerfile.spark ../docker
+   ```
+2. Start the HDFS and Spark cluster.
+
+   ```bash
+   $ docker-compose up -d
+   ```
+
+   After the HDFS cluster has started, you should be able to navigate to the
+   web UI at [localhost:50070](http://localhost:50070) to view the HDFS dashboard.
+
+   After the Spark cluster has started, you should be able to navigate to the
+   web UI at [localhost:9090](http://localhost:9090) to view the Spark dashboard.
```

Comment:
It seems that the port `9090` is not mentioned in Spark's official documents. How about using `8080` in this document and example? We may add a FAQ section to resolve questions about port exposure.